src/features/context-menu/components/ContextMenuComponent.js

```diff
@@ -210,13 +210,13 @@ class ContextMenu extends Component {
   componentDidMount() {
     document.addEventListener('focusin', this.onFocusChanged);
     document.addEventListener('keydown', this.onGlobalKey);
-    document.addEventListener('click', this.onGlobalClick);
+    document.addEventListener('mousedown', this.onGlobalClick);

```

Comment:
Can we rename the method name `onGlobalClick` since the event changed?

Reply:
Sure.